Xamarin.Forms.Platform.MacOS/Controls/FormsBoxView.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using AppKit;
+using CoreGraphics;
+
+namespace Xamarin.Forms.Platform.macOS.Controls
+{
+	internal class FormsBoxView : NSView
+	{
+		NSColor _colorToRenderer;
+
+		nfloat _topLeft;
+		nfloat _topRight;
+		nfloat _bottomLeft;
+		nfloat _bottomRight;
+
+		public override void DrawRect (CGRect dirtyRect)
+		{
+			_colorToRenderer.SetFill ();
```

Comment:
Where is this init? it throws NRE

Reply:
I think I should have lost something at some point respect to the original branch: https://github.com/XamarinFormsCommunity/Xamarin.Forms/blob/feature/1709-rounded-boxview/Xamarin.Forms.Platform.MacOS/Renderers/BoxViewRenderer .cs
Fixed.